### PR TITLE
docs: Define the term "open source"

### PR DESCRIPTION
**Pull request by blacklight447**
Relevant discussion:https://discuss.privacyguides.net/t/what-does-privacy-guides-see-as-opensource/20230

### CODE REVIEW

**Comment by blacklight447** [comment]
If we move forward with this, we should overlook all pages with opensource requirements and check the currently listed software

File: docs/about/criteria.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: General Criteria
 Below are some general priorities we consider for all submissions to Privacy Guides. Each category will have additional requirements for inclusion.
 
 - **Security**: Tools should follow security best-practices wherever applicable.
-- **Source Availability**: Open-source projects are generally preferred over equivalent proprietary alternatives.
+- **Source Availability**: Open-source projects are generally preferred over equivalent proprietary alternatives. Our definition of Open-source follows the [OSI definition](https://opensource.org/osd). Licenses not under the OSI are allowed as long as they are compatible with the OSI definition. The Open-source part is only mandatory for pages with "Open-source" as a minimum requirement.
```
Comment (jonaharagon) [inline docs/about/criteria.md:8]: We discussed _not_ making this change in https://github.com/privacyguides/privacyguides.org/pull/2727#discussion_r1726180406

Comment (jonaharagon) [reply docs/about/criteria.md:8]: What I was more saying in the comment I linked is that *regardless* of the final decision, I'm not sure this is the best place on the website to expand on these details.

Comment (friadev) [reply docs/about/criteria.md:8]: Where's a better place?

**Comment by jonaharagon** [comment]
I'm going to close this because this general page isn't where we should add technical/minor details.

@friadev (re: https://github.com/privacyguides/privacyguides.org/pull/2732#discussion_r1837187399) A better place would be https://github.com/privacyguides/privacyguides.org/pull/2727#discussion_r1726209176 for example.

**Pull request closed without merging**